dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/Run.java

```diff
@@ -1006,6 +1010,8 @@ protected int runSpringBoot() throws Exception {
         eq.files = this.files;
         eq.name = this.name;
         eq.gav = this.gav;
+        eq.repositories = this.repositories;
+        eq.camelSpringBootVersion = this.camelSpringBootVersion;
```

Comment:
this breaks as the camelSpringBootVersion has been set a bit higher up in the code to camelVersion